.coveragerc

```diff
@@ -1,7 +1,11 @@
 # .coveragerc to control coverage.py
 [run]
+branch = True
 omit =
```

Comment:
Let me know what other files to omit from the coverage reports

Reply:
For now I think simulator interfaces that can't be run from the test suite should probably be excluded (until we get the CI set up to test them).

Reply:
Ack! `src/scenic/simulators/*` has been excluded from the overall code coverage report!